tests/TestRunner/main.cpp

```diff
@@ -30,7 +30,12 @@ static constexpr const wchar_t* g_applications[] =
     L"CompositionTest_8wekyb3d8bbwe!Fixed",
     L"FileSystemTest_8wekyb3d8bbwe!Fixed",
     L"LongPathsTest_8wekyb3d8bbwe!Fixed",
-    L"WorkingDirectoryTest_8wekyb3d8bbwe!Fixed"
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTNoScripts",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTOnlyStart",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTOnlyEnd",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTBothStartingFirst",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTBothEndingFirst",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTScriptWithArg"

```

Comment:
Nothing actually validates the script outputs, right? So is the idea here just that if the exit code is zero we assume that the scripts ran successfully? Perhaps considering doing something in the scripts that has a side effect that's observable that the test executable can check for and fail if it wasn't done. That'll only really test the 'start' script, but perhaps that's good enough

Reply:
You are correct.  There is no way to tell if there was an error inside a script. 

I decided not to check for errors inside scripts because the script authors might expect an error to happen.  This would happen in case the script writes to a file that exists and if the file exists to not do anything.  

Because people can expect errors to happen in a script I don't test for it.

Reply:
Okay, I'll rephrase it this way: _what are you actually testing here_? I figured that the idea was to test that the scripts were actually run, but they don't appear to be doing that. AFAICT they just run and succeed pretty unconditionally. So, what's the point of even having these tests then?

Reply:
To test the config file.  I want to make sure each script will run correctly based on different configuration files.

Example.  Having an ending script but no starting script.

Reply:
> I want to make sure each script will run correctly

Right. I'm saying that you are _not_ doing that. E.g. if the script configuration were ignored (or malformed, etc.) these would still pass. All that it validates is that the executable is run; there's no validation that any scripts were actually run AFAICT

Reply:
I see.  You are correct.  I'm not testing if any scripts were ran.

I'll make the scripts write a sentinel file and have the tests verify that the file exists.